server/controllers/Events/resolver.ts

```diff
@@ -56,7 +58,7 @@ export class EventResolver {
       if (!venue) throw new Error('Cant find venue');
       event.venue = venue;
     }
-
+    if (event.url) this.validateEventUrl(event.url);
```

Comment:
Nit: single line if statements are discouraged due to their brittleness.

```suggestion
    if (event.url) {
      this.validateEventUrl(event.url);
    }
```

Reply:
(same change to where this is done above)

Reply:
They're gone now that the validation is gone but I was following the convention already established in the file. If we don't want single life ifs maybe we should update the lint settings